test/system/250-systemd.bats

```diff
@@ -479,4 +479,41 @@ $name stderr" "logs work with passthrough"
     run_podman generate --help
     is "$output" ".*\[DEPRECATED\] Generate systemd units"
 }
+
+@test "podman network reload on firewall-cmd --reload" {
+    setup_firewalld_services
```

Comment:
This test must be skipped as rootless and when firewalld is not installed or not active.

But in general I am not sure we really want to test this in systems test, reloading the firewalld could have negative consequences for other applications as well outside of the tests. Given the high amount of different environments these tests are run I don't think it is a good idea to do it.